build.proj

```diff
@@ -12,6 +12,11 @@
 
     <NoRunArg Condition=" '$(OS)' == 'Windows_NT' ">-NoRun</NoRunArg>
     <NoRunArg Condition=" '$(OS)' != 'Windows_NT' ">--norun</NoRunArg>
+    
+    <CR>%0D</CR>
+    <LF>%0A</LF>
+    <NewLine Condition=" '$(OSName)' == 'win' ">$(LF)</NewLine>
+    <NewLine Condition=" '$(OSName)' != 'win' ">$(CR)$(LF)</NewLine>
```

Comment:
Isn't this backwards?  Windows has CRLF, non-Windows uses just LF.

---
In reply to: [69179485](https://github.com/dotnet/cli/pull/3737#discussion_r69179485) [](ancestors = 69179485)